src/status_im/translations/en.cljs

```diff
@@ -50,9 +50,13 @@
 
    :help-improve                         "Help improve Status\nby sharing usage patterns"
    :help-improve?                        "Help improve Status?"
-   :help-improve-description             "We collect anonymous data to help improve Status."
-   :share-usage-data                     "Share usage"
-   :dont-want-to-share                   "No, I don't want to share"
+   :help-improve-description             "We collect anonymous data to understand how Status is used and inform future development. Whilst we appreciate your help to make Status better, this is an optional choice and can be changed at any time."
+   :learn-what-we-collect-link           "Learn more about what we collect"
+   :share-usage-data                     "Share data"
+   :dont-want-to-share                   "Do not share"
+   :confirmation-title                   "Share data with Status?"
+   :confirmation-text                    "Are you sure you want to share anonymous data with Status?"
+   :confirmation-action                  "Share"
```

Comment:
Any reason why this is RED in the alert?

Reply:
OK this is because `utils/show-confirmation` applies the IOS-specific style "destructive" to the confirm button:
```
           ;; Styles are only relevant on iOS. On Android first button is 'neutral' and second is 'positive'
<snip>
                    {:text                (or confirm-button-text "OK")
                     :onPress             on-accept
                     :style               "destructive"
                     :accessibility-label :confirm-button})))))
```
Documented here: https://developer.apple.com/documentation/uikit/uialertactionstyle/1620089-destructive
Seems likely that style is in red!

Reply:
Can we use the neutral style?

Reply:
Sure, will parameterize the behaviour in `utils/show-confirmation` so we can override it here.

Reply:
Done!